lib/parse.js

```diff
@@ -2109,7 +2109,7 @@ function parse($TEXT, options) {
         return function_(AST_Accessor, is_generator, is_async);
     });
 
-    var object_or_object_destructuring_ = embed_tokens(function() {
+    var object_or_destructuring_ = embed_tokens(function object_or_destructuring_() {
```

Comment:
There will be one tiny issue to watch for, i.e. if I later copy and paste some code that calls `object_or_destructuring_()` into this function, then it would now be calling this directly as opposed to `embed_tokens()`-wrapped version.